.github/workflows/build_and_test.yml

```
@@ -1054,13 +1099,48 @@ jobs:
         cd OpenDDS
         . setenv.sh
         tools/scripts/show_build_config.pl
-    - name: build and run CMake tests
+    - name: run OpenDDS tests
       shell: bash
       run: |
         cd OpenDDS
         . setenv.sh
-        cd tests/cmake
-        perl run_ci_tests.pl --compiler=clang++-5.0 --cxx-standard 11
+        mkdir ${{ github.job }}_autobuild_workspace
+        cd ${{ github.job }}_autobuild_workspace
+        echo using commit $TRIGGERING_COMMIT for SHA
+        echo $TRIGGERING_COMMIT >> ./SHA
+        cat > config.xml <<EOF
+        <autobuild>
+        <configuration>
+        <variable name="log_file" value="output.log"/>
+        <variable name="log_root" value="$DDS_ROOT/${{ github.job }}_autobuild_workspace"/>
+        <variable name="project_root" value="$DDS_ROOT"/>
+        <variable name="root" value="$DDS_ROOT/${{ github.job }}_autobuild_workspace"/>
+        <variable name="junit_xml_output" value="Tests"/>
+        </configuration>
+        <command name="log" options="on"/>
+        <command name="print_os_version"/>
+        <command name="print_env_vars"/>
+        <command name="print_ace_config"/>
+        <command name="print_make_version"/>
```

```suggestion
        <command name="print_cmake_version"/>
        <command name="print_make_version"/>
```

I forgot to do this when I added CMake to the actions before, but could you do this for the places where I added `--cmake` to auto_run_tests as well as the places your adding.